File: 7KYU/Make a function that does arithmetic!/solution.c
#include <string.h>

double arithmetic(double a, double b, const char* operator) {
  if(strcmp(operator, "add")==0){
    return a+b; 
  }
  if(strcmp(operator, "subtract")==0){
    return a-b;
  }
  if(strcmp(operator, "multiply")==0){
    return a*b;
  }
  if(strcmp(operator, "divide")==0){
    return a/b;
    
  }
  
}

File: 7KYU/Make a function that does arithmetic!/solution.go
package kata

func Arithmetic(a int, b int, operator string) int {
	if operator == "add" {
		return a + b
	}
	if operator == "subtract" {
		return a - b
	}
	if operator == "multiply" {
		return a * b
	}
	if operator == "divide" {
		return a / b
	}
	panic("invalid")
}
